src/invidious/channels/community.cr

```diff
@@ -125,49 +125,11 @@ def fetch_channel_community(ucid, continuation, locale, format, thin_mode)
 
               if attachment = post["backstageAttachment"]?
                 json.field "attachment" do
-                  json.object do
-                    case attachment.as_h
-                    when .has_key?("videoRenderer")
-                      attachment = attachment["videoRenderer"]
-                      json.field "type", "video"
-
-                      if !attachment["videoId"]?
-                        error_message = (attachment["title"]["simpleText"]? ||
-                                         attachment["title"]["runs"]?.try &.[0]?.try &.["text"]?)
-
-                        json.field "error", error_message
-                      else
-                        video_id = attachment["videoId"].as_s
-
-                        video_title = attachment["title"]["simpleText"]? || attachment["title"]["runs"]?.try &.[0]?.try &.["text"]?
-                        json.field "title", video_title
-                        json.field "videoId", video_id
-                        json.field "videoThumbnails" do
-                          Invidious::JSONify::APIv1.thumbnails(json, video_id)
-                        end
-
-                        json.field "lengthSeconds", decode_length_seconds(attachment["lengthText"]["simpleText"].as_s)
-
-                        author_info = attachment["ownerText"]["runs"][0].as_h
-
-                        json.field "author", author_info["text"].as_s
-                        json.field "authorId", author_info["navigationEndpoint"]["browseEndpoint"]["browseId"]
-                        json.field "authorUrl", author_info["navigationEndpoint"]["commandMetadata"]["webCommandMetadata"]["url"]
-
-                        # TODO: json.field "authorThumbnails", "channelThumbnailSupportedRenderers"
-                        # TODO: json.field "authorVerified", "ownerBadges"
-
-                        published = decode_date(attachment["publishedTimeText"]["simpleText"].as_s)
-
-                        json.field "published", published.to_unix
-                        json.field "publishedText", translate(locale, "`x` ago", recode_date(published, locale))
-
-                        view_count = attachment["viewCountText"]?.try &.["simpleText"].as_s.gsub(/\D/, "").to_i64? || 0_i64
-
-                        json.field "viewCount", view_count
-                        json.field "viewCountText", translate_count(locale, "generic_views_count", view_count, NumberFormatting::Short)
```

Comment:
In order to not break compatibility, `viewCountText` needs to be added to `SearchVideo#to_json`